services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -36,6 +37,8 @@
 # the maximum number of unique values in a string column so that it is considered to be a category,
 # otherwise it's treated as a string
 MAX_NUM_STRING_LABELS = 30
+# datasets.ClassLabel feature uses -1 to encode `no label` value
+NO_LABEL_VALUE = -1
```

Comment:
https://github.com/huggingface/datasets/blob/faedc670ca896584d0f8edcb1fd9c13d1d6cc903/src/datasets/features/features.py#L1077

not exported as a constant, so: ok to define it here instead